debezium-embedded/src/main/java/io/debezium/embedded/EmbeddedEngine.java

```diff
@@ -760,6 +760,14 @@ public OffsetStorageReader offsetStorageReader() {
                         connectorCallback.ifPresent(DebeziumEngine.ConnectorCallback::taskStarted);
                     }
                     catch (Throwable t) {
+                        // Clean-up allocated resources
+                        try {
+                            LOGGER.debug("Stopping the task");
+                            task.stop();
+                        }
+                        catch (Throwable tstop) {
+                            LOGGER.info("Error while trying to stop the task");
```

Comment:
```suggestion
                            LOGGER.error("Error while trying to stop the task");
```

Same for the other one perhaps?

Reply:
I intentionally set the `info` level as this is error that is just ignored